azure-mgmt-containerinstance/HISTORY.rst

```diff
@@ -0,0 +1,9 @@
+.. :changelog:
+
+Release History
+===============
+
+0.1.0 (1970-01-01)
```

Comment:
I know, I change it the day of release, and since we are not releasing today :). That's the default date in the HISTORY.txt template.